core/src/filed/fd_plugins.cc

```diff
@@ -742,11 +742,13 @@ int PluginSave(JobControlRecord* jcr, FindFilesPacket* ff_pkt, bool)
             Jmsg1(jcr, M_FATAL, 0,
                   T_("Command plugin \"%s\": startBackupFile failed.\n"), cmd);
             goto bail_out;
+          case bRC_More:
+            Dmsg0(debuglevel, "Plugin returned bRC_More, continue\n");
+            goto fun_end;
```

Comment:
I can't tell you.
But I can tell you that plugins are using it and that you get a warning on your job unless we apply that change :)

Reply:
One way to prevent this is to only set a flag when an override was detected (maybe also save information related to the overwrite; i.e. which option was overridden).  Then we can emit a warning if we catch `bEventStartBackupJob`.